engine/movie/credits.asm

```diff
@@ -85,7 +85,7 @@ Credits::
 	xor a
 	ldh [hBGMapMode], a
 	ld [wCreditsPos], a
```

Comment:
```suggestion
	ld [wCreditsPos + 0], a
```
I don't remember if we settled to do this for arrays, but if we did please change it.

Reply:
I don't think we settled on that, or at least, the other two spots which load to `wCreditsPos` have `[wCreditsPos]` and `[wCreditsPos + 1]`.